.github/workflows/ci.yml

```
@@ -3,38 +3,83 @@
   detect-changes:
     runs-on: ubuntu-latest
     outputs:
       auth: ${{ steps.changes.outputs.auth }}
-      subscription: ${{ steps.changes.outputs.subscription }}
+      any: ${{ steps.changes.outputs.any }}
     steps:
       - uses: actions/checkout@v4
-      - uses: dorny/paths-filter@v2
-        id: changes
+      - id: changes
+        uses: dorny/paths-filter@v2
```

actionlint (1.7.7)

the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)